guides/v2.3/graphql/search-pagination.md

```diff
@@ -47,17 +47,16 @@ Magento GraphQL clause | SQL equivalent
 `gt: "value"`	| <code><i>field</i> > 'value'</code>
 `gteq: "value"`	| <code><i>field</i> >= 'value'</code>
 `lteq: "value"`	| <code><i>field</i> <= 'value'</code>
```

Comment:
`moreq` is still supported, according to https://github.com/magento/graphql-ce/blob/2.3-develop/app/code/Magento/GraphQl/etc/schema.graphqls  Back out this edit.

Reply:
input FilterTypeInput @doc(description: "FilterTypeInput specifies which action will be performed in a query ") {
    eq: String @doc(description: "Equals")
    finset: [String] @doc(description: "Find in set. The value can contain a set of comma-separated values")
    from: String @doc(description: "From. Must be used with 'to'")
    gt: String @doc(description: "Greater than")
    gteq: String @doc(description: "Greater than or equal to")
    in: [String] @doc(description: "In. The value can contain a set of comma-separated values")
    like: String @doc(description: "Like. The specified value can contain % (percent signs) to allow matching of 0 or more characters")
    lt: String @doc(description: "Less than")
    lteq: String @doc(description: "Less than or equal to")
    moreq: String @doc(description: "More than or equal to")
    neq: String @doc(description: "Not equal to")
    notnull: String @doc(description: "Not null")
    null: String @doc(description: "Is null")
    to: String@doc(description: "To. Must be used with 'from'")
    nin: [String] @doc(description: "Not in. The value can contain a set of comma-separated values")
}
@keharper sir,

Is there any reason supporting `moreq` since there is already `gteq` available?

Reply:
i re added it.